### PR TITLE
SISRP-52459: Surface Instruction Mode Within Class Information Card

### PR DESCRIPTION
**Pull request by developish**
https://jira-secure.berkeley.edu/browse/SISRP-52459

### CODE REVIEW

File: app/models/hub_edos/classes_api/v1/section.rb [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+    found&.description
+  end
+
+  SectionAttribute = Struct.new(:data) do
+    def code
+      data['attribute']['code']
+    end
+
+    def value
```
Comment (redconfetti) [inline app/models/hub_edos/classes_api/v1/section.rb:68]: @developish - Maybe we can update this to use `HubEdos::Common::Reference::Descriptor` from #658 at some point in the future.

File: app/models/my_academics/semesters.rb [hunk 2]
```diff
@@ -155,6 +155,7 @@ def merge_semesters_count(data)
     def process_enrollments(enrollment_term, term_id)
       filtered_enrollment_term = filter_enrollments(enrollment_term)
       filtered_enrollment_term.map do |enrollment|
+        # mapped_enrollment = CourseEnrollmentInfo.new(enrollment).as_json
```
Comment (redconfetti) [inline app/models/my_academics/semesters.rb:158]: @developish - Can we get rid of this?

Comment (developish) [reply app/models/my_academics/semesters.rb:158]: You bet!

File: src/react/components/VerticalSpacer.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import React from 'react'
+
+export default function VerticalSpacer() {
+  return <div style={{ marginTop: `15px` }} />
+}
```
Comment (redconfetti) [inline src/react/components/VerticalSpacer.js:5]: @developish - Please add line, Github no likey

File: src/react/components/_academics/ClassInformationCard/ClassInformationContainer.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { useLocation } from 'react-router-dom';
+import axios from 'axios';
+
+import ClassInformationCard from './ClassInformationCard';
+
+const fetchCourseSections = async (termId, courseId, sectionIds) => {
+  const response = await axios.get(
+    `/api/my/academics/terms/${termId}/courses/${courseId}/sections`,
```
Comment (redconfetti) [inline src/react/components/_academics/ClassInformationCard/ClassInformationContainer.js:11]: Sweet!

File: src/react/components/VerticalSpacer.js (hunk 3 above)
Comment (developish) [reply src/react/components/VerticalSpacer.js]: Done!

**Pull request merged**